lib/browser/guest-view-manager.js

```diff
@@ -184,6 +184,32 @@ const attachGuest = function (embedder, elementInstanceId, guestInstanceId, para
     disableBlinkFeatures: params.disableblinkfeatures
   }
 
+  // parse the 'webpreferences' attribute string, if set
+  // this uses the same parsing rules as window.open uses for its features
```

Comment:
I had the same thought, but I did find a way to make it work, that I think is pretty clean. I just need to solve the require() situation.